### PR TITLE
[Teach-298] Create section - Update dialog styling

### PR DESCRIPTION
**Pull request by levadadenys**
### [Teach-298] Create section - Update dialog styling
Restyling(Visual appearance of pr) is already approved by Molly Peredo

Before:
image

After:

image

image

image

## Links

- jira ticket: [Teah-298](https://codedotorg.atlassian.net/browse/TEACH-298?atlOrigin=eyJpIjoiNTc0M2QwMmJmODNmNDNiMjhlZTQyZDQyMGFkMGZmYzQiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/teacherDashboard/LoginTypePicker.jsx [hunk 1]
```diff
@@ -98,22 +99,30 @@ class LoginTypePicker extends Component {
...
+        paddingTop: '8px',
+        borderTop: `1px solid ${color.neutral_dark}`
       }
     };
 
     return (
       <div style={style.container}>
-        <Heading1>{title}</Heading1>
-        <Heading2>{i18n.addStudentsToSectionInstructionsUpdated()}</Heading2>
+        <Heading3>{title}</Heading3>
```
Comment (rshipp) [inline apps/src/templates/teacherDashboard/LoginTypePicker.jsx:124]: i can't find a definitive answer on what starting heading level inside a modal is best for screenreaders, but i would guess that h2 or h1 would be easier to navigate. would the `Heading3`s in here work as `Heading2` or `Heading1` with a class to adjust the styling?

that said, this modal already has other major accessibility issues, so I wouldn't consider this a blocking comment.

Comment (levadadenys) [reply apps/src/templates/teacherDashboard/LoginTypePicker.jsx:124]: Not sure why h3 should be a problem for screenreader. h3 is still a header so I don't see any semantic issues here, correct me if I'm wrong. In my opinion it's better to stick with one type-specs that we agreed and put in [this file](https://staging.code.org/rebrand/rebrand-type-spec#)

Comment (levadadenys) [reply apps/src/templates/teacherDashboard/LoginTypePicker.jsx:124]: As an option - we can discuss it with designer, Molly Peredo (Looks like she doesn't have a githun account)

Comment (rshipp) [reply apps/src/templates/teacherDashboard/LoginTypePicker.jsx:124]: this page has some best practices: https://usability.yale.edu/web-accessibility/articles/headings#using. ideally, headings should only be chosen for their semantic meaning, and not based on style. it sounds like there's larger design questions here that make this outside the scope of this PR, so I can ask Molly about this separately. thanks for that link!

File: apps/src/templates/teacherDashboard/ParticipantTypePicker.jsx [hunk 2]
```diff
@@ -43,14 +44,16 @@ export default class ParticipantTypePicker extends Component {
...
+          {i18n.professionalLearningParticipantQuestion()}
+        </p>
         <div style={style.scroll}>
           <CardContainer>
-            {this.props.availableParticipantTypes.map((type, index) => (
+            {this.props.availableParticipantTypes.map(type => (
               <ParticipantTypeCard
                 onClick={setParticipantType}
-                key={index}
+                key={type}
```
Comment (rshipp) [inline apps/src/templates/teacherDashboard/ParticipantTypePicker.jsx:56]: nice catch!

File: apps/src/templates/teacherDashboard/LoginTypeCard.jsx [hunk 3]
```diff
@@ -15,9 +15,11 @@ class LoginTypeCard extends Component {
     hover: false
   };
 
-  toggleHover = () => {
-    this.setState({hover: !this.state.hover});
-  };
+  // Toggle hover is not working properly when component is rendered with mouse already 'over' the component.
+  // In order for hover to work properly - set on/off hover implicitly.
```
Comment (rshipp) [inline apps/src/templates/teacherDashboard/LoginTypeCard.jsx:19]: thanks for this comment!

**Review by rshipp — APPROVED** [review]
started a convo with Molly about my heading questions, but I don't think that should block this PR. the rest looks good to me!

File: apps/src/lib/ui/Headings.jsx [hunk 4]
```diff
@@ -46,9 +46,9 @@ export class Heading2 extends Component {
 
 export const h3Style = {
   ...baseHeadingStyle,
-  fontFamily: '"Gotham 5r", sans-serif',
-  fontSize: 16,
-  lineHeight: '24px'
+  fontFamily: '"Barlow Semi Condensed Semibold", sans-serif',
```
Comment (bethanyaconnor) [inline apps/src/lib/ui/Headings.jsx:49]: Just checking: will this affect any other components unexpectedly?

**Review by bethanyaconnor — APPROVED** [review]
🎉

File: apps/src/lib/ui/Headings.jsx (hunk 4 above)
Comment (levadadenys) [reply apps/src/lib/ui/Headings.jsx:49]: I've checked, it should only affect PR

**Pull request merged**